main.go

```diff
@@ -663,8 +740,14 @@ func configureSite(ctx *cli.Context, siteNum int, siteStrs []string, healthCheck
 
 		proxy := &httputil.ReverseProxy{
 			Director: func(r *http.Request) {
-				r.Header.Add("X-Forwarded-Host", r.Host)
-				r.Header.Add("X-Real-IP", r.RemoteAddr)
+				if strings.HasPrefix(r.RemoteAddr, "localhost") || strings.HasPrefix(r.RemoteAddr, "127.0.0.1") {
```

Comment:
```suggestion
				if strings.HasPrefix(r.RemoteAddr, "localhost") || strings.HasPrefix(r.RemoteAddr, "127.0.0.1") {
```

This is not complete there are other local IPs such as the entire 127.0.0.x range.

```go
func IsLoopback(addr string) bool {
	host, _, err := net.SplitHostPort(addr)
	if err != nil {
		host = addr
	}
	if host == "localhost" || host == "127.0.0.1" || host == "::1" {
		return true
	}
        return net.ParseIP(host).IsLoopback()
}
```

For the most it will always be either 127.0.0.1 or ::1 - however this code would catch other random users such as running many sidekicks on 127.0.0.2, 127.0.0.3 ...

Reply:
Ack. Thanks for this.